packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -65,6 +93,68 @@ export const PaymentUpdateForm = ({
     resetUpdatePaymentIsSuccess,
   ]);
 
+  const {
+    exp_month,
+    exp_year,
+    payment_provider,
+    paypal_payment_error,
+  } = customer;
+
+  const actionButton = (
+    <ActionButton
+      {...{ customer, onRevealUpdateClick, revealFixPaymentModal }}
+    />
+  );
+
+  const errorAlertBarContent = (
+    <Localized
+      id="sub-route-invalid-payment-alert"
+      elems={{ div: actionButton }}
+    >
+      <span>
+        Invalid payment information, there is an error with your account.{' '}
+        {actionButton}
+      </span>
+    </Localized>
```

Comment:
Never seen this `elems` attribute. Is this why you have the `<div>` inside the `main.ftl` file? What is this doing exactly?

Reply:
https://github.com/projectfluent/fluent.js/wiki/Localized#markup